finetune/base.py

```diff
@@ -177,6 +177,24 @@ def finetune(self, Xs, Y=None, batch_size=None):
 
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
+            if self.config.prefit_init:
+                tf.logging.info("Starting pre-fit initialisation...")
+                num_layers_trained = self.config.num_layers_trained
+                self.config.num_layers_trained = 0
+                estimator.train(train_input_fn, hooks=train_hooks, steps=num_steps)
+                self.config.num_layers_trained = num_layers_trained
+                self.saver.variables = {k: v for k, v in self.saver.variables.items() if "adam" not in k and "global_step" not in k}
+                for weight in self.saver.variables:
+                    if weight.startswith("model/target/"):
+                        w = self.saver.variables[weight]
+                        shape = w.shape
```

Comment:
Feels weird to alias this to save two characters but whatevs.  Some weird spacing w/ len(shape) below.